commands/release/release.go

```diff
@@ -73,12 +73,19 @@ var (
 	`
 )
 
+var shouldBumpMajorVersion bool
 var ReleaseCmd = &cobra.Command{
 	Use:   releaseCmdStr,
 	Short: "Cuts a new release on the repo",
+	Long:  "Cuts a new release on a Kurtosis Repo. This command is intended to be ran in a github action and requires a release token to authenticate pushes to master.",
```

Comment:
Can we appease my OCD and proper-case it to "Github"? 😅

Reply:
i'll think about it :)